metadata_service/config.py

```diff
@@ -114,3 +116,39 @@ class LocalConfig(LocalGremlinConfig, Config):
         'title': 'Metadata Service',
         'uiversion': 3
     }
+
+
+# The databuilder expects this to be False currently. We are defaulting to true because the testing expects this
+if bool(distutils.util.strtobool(os.environ.get('IGNORE_NEPTUNE_SHARD', 'False'))):
```

Comment:
This needs to be at the base later instead of the gremlin proxy because once get_shard is called the shard_set_explicitly no longer works.